### PR TITLE
OpenScanHub integration fixes

### PR DESCRIPTION
**Pull request by lbarcziova**

### CODE REVIEW

File: files/install-deps-worker.yaml [hunk 1]
```diff
@@ -44,6 +44,7 @@
           - epel-rpm-macros
           # for the `pkg_tool` switch to allow centpkg
           - centpkg
+          - osh
```
Comment (siteshwar) [inline files/install-deps-worker.yaml:47]: Should not this be `osh-cli`?

File: packit_service/worker/handlers/copr.py [hunk 2]
```diff
@@ -643,12 +645,13 @@ def get_base_srpm_model(
             target="fedora-rawhide-x86_64",
             status=BuildStatus.success,
         )
-        if not builds:
+
+        try:
+            return next(iter(builds)).get_srpm_build()
+        except StopIteration:
```
Comment (mfocko) [inline packit_service/worker/handlers/copr.py:651]: You got this in Sentry?

File: files/install-deps-worker.yaml [hunk 3]
```diff
@@ -44,6 +44,7 @@
           - epel-rpm-macros
           # for the `pkg_tool` switch to allow centpkg
           - centpkg
+          - osh-cli
```
Comment (mfocko) [inline files/install-deps-worker.yaml:47]: Is it packaged for CentOS 9 Stream too?

**Review by mfocko — APPROVED** [review]

File: files/install-deps-worker.yaml (hunk 3 above)
Comment (mfocko) [reply files/install-deps-worker.yaml:47]: I'll answer myself after checking :D no, but there are builds in Copr `@openscanhub/devel`

Comment (mfocko) [reply files/install-deps-worker.yaml:47]: But I don't see CLI in there :eyes:

```
osh-hub 1.0.0.20240704.103731.g4de668b
osh-common 1.0.0.20240704.103731.g4de668b
osh-worker 1.0.0.20240704.103731.g4de668b
osh-hub-conf-devel 1.0.0.20240704.103731.g4de668b
osh-worker-conf-devel 1.0.0.20240704.103731.g4de668b
osh-client 1.0.0.20240704.103731.g4de668b
```

Comment (mfocko) [reply files/install-deps-worker.yaml]: pushed the fix, please have a look

Comment (lbarcziova) [reply files/install-deps-worker.yaml]: thanks for having a look into this, LGTM!

**Pull request merged**